tachyon/math/polynomials/univariate/univariate_evaluations.h

```diff
@@ -199,9 +191,22 @@ class UnivariateEvaluations final
 
  private:
   friend class internal::UnivariateEvaluationsOp<F, MaxDegree>;
+  // NOTE(chokobole): Below doesn't work since we have a code that tries to
+  // create an evaluations from a domain whose field is |RationalField<F>|.
+  // friend class UnivariateEvaluationDomain<F, MaxDegree>;
```

Comment:
```Suggestion
  // NOTE(chokobole): Commented code below doesn't work since we have code 
  // that tries to create evaluations from a domain whose field is 
  // |RationalField<F>|.
  // friend class UnivariateEvaluationDomain<F, MaxDegree>;
```